<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Communication</class>
 <widget class="QMainWindow" name="Communication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>681</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(33, 36, 255);</string>
    </property>
    <property name="text">
     <string>通信参数设置</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="QRcodeGroupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>580</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>二维码扫描仪：</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>25</y>
       <width>54</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>串口号：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="QRcodeSerialPortComboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>25</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>25</y>
       <width>54</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="QRcodeBuadrateComboBox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>25</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>56000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>128000</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="QRcodeDataBitComboBox">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>25</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>25</y>
       <width>54</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>数据位：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="QRcodeStopBitComboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>65</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>65</y>
       <width>54</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>停止位：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="QRcodeParityBitComboBox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>65</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>NONE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ODD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>EVEN</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MARK</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SPACE</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>65</y>
       <width>54</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>校验位：</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="cameraGroupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>580</width>
      <height>85</height>
     </rect>
    </property>
    <property name="title">
     <string>拍照系统：</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="systemGroupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>580</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>盖章系统：</string>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>65</y>
       <width>54</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>停止位：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>25</y>
       <width>54</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="systemDataBitComboBox">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>25</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>25</y>
       <width>54</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>串口号：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="systemParityBitComboBox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>65</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>NONE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ODD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>EVEN</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MARK</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SPACE</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="systemStopBitComboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>65</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="systemBuadrateComboBox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>25</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>56000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>128000</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>25</y>
       <width>54</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>数据位：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="systemSerialPortComboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>25</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>65</y>
       <width>54</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>校验位：</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="connectPushButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>415</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>连 接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetPushButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>415</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>重 置</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
